frontend/routes/Wifi.jsx

```diff
@@ -10,44 +28,268 @@ export class Component extends React.Component {
+      is_num = is_num || element.numeric
+  });
+
+  if(is_num){
+    if (parseInt(b[orderBy]) < parseInt(a[orderBy])) {
+      return -1;
+    }
+    if (parseInt(b[orderBy]) > parseInt(a[orderBy])) {
+      return 1;
+    }
+    return 0;
+  }
+  else{
+    return a[orderBy] < b[orderBy] ? -1 : a[orderBy] > b[orderBy];    
+  }
+
+
+}
+
+function getComparator(order, orderBy) {
+  
+  return order === 'desc'
+    ? (a, b) => descendingComparator(a, b, orderBy)
+    : (a, b) => -descendingComparator(a, b, orderBy);
+}
+
+function stableSort(array, comparator) {
+  const stabilizedThis = array.map((el, index) => [el, index]);
+  stabilizedThis.sort((a, b) => {
+    const order = comparator(a[0], b[0]);
+    if (order !== 0) return order;
+    return a[1] - b[1];
+  });
+  return stabilizedThis.map((el) => el[0]);
+}
+
+function EnhancedTable(props) {
+  const classes = useStyles();
+  const [order, setOrder] = React.useState('asc');
+  const [orderBy, setOrderBy] = React.useState(headCells[0].id);
+  const [dense, setDense] = React.useState(false);
+  const {rows} = props;
+
+  const handleRequestSort = (event, property) => {
+    const isAsc = orderBy === property && order === 'asc';
+    setOrder(isAsc ? 'desc' : 'asc');
+    setOrderBy(property);
+  };
+
+
+  const handleChangeDense = (event) => {
+    setDense(event.target.checked);
+  };
+
+  return (
+    <div className={classes.root}>
+      <Paper className={classes.paper}>
+        <EnhancedTableToolbar />
+        <TableContainer>
+          <Table
+            className={classes.table}
+            aria-labelledby="tableTitle"
+            size={dense ? 'small' : 'medium'}
+            aria-label="enhanced table"
+          >
+            <EnhancedTableHead
+              classes={classes}
+              order={order}
+              orderBy={orderBy}
+              onRequestSort={handleRequestSort}
+            />
+            <TableBody>
+              {stableSort(rows, getComparator(order, orderBy))
+                .map((row, index) => {
+                  const labelId = `enhanced-table-checkbox-${index}`;
+                  return (
+                    <TableRow
+                      hover
+                      tabIndex={-1}
+                      key={index}                 
+                      >               
+                      {
+                        headCells.map((element)=>{
+                        return <TableCell key={element.id} align="center">{row[element.id]}</TableCell>
+                        })
+                      }
+                    </TableRow>
+                  );
+                })}
+
+              {/* {emptyRows > 0 && (
+                <TableRow style={{ height: (dense ? 33 : 53) * emptyRows }}>
+                  <TableCell colSpan={headCells.length} />
+                </TableRow>
+              )} */}
+
+            </TableBody>
+          </Table>
+        </TableContainer>
+      </Paper>
+      <FormControlLabel
```

Comment:
I don't see the value in adding this toggle here you can remove it for now, maybe we can globally add such a toggle to a settings in the future.